.tool-versions

```diff
@@ -1,4 +1,4 @@
 # this file gets used by asdf and uses a symbolic version
 # create a symlink named 11 in ~/.asdf/installs/java to the 11 version you want to use
 # then run `asdf reshim java`
-java 11
+java temurin-11.0.24+8
```

Comment:
not an `asdf` expert, but just specifying `11` didn't work for me (not found error)